sources/packages/backend/apps/queue-consumers/src/processors/schedulers/sfas-integration/_tests_/sfas-integration.scheduler.e2e-spec.ts

```diff
@@ -238,6 +242,203 @@ describe(describeProcessorRootTest(QueueNames.SFASIntegration), () => {
     },
   );
 
+  it(
+    "Should not add SFAS individual data records and display errors when importing invalid data from SFAS " +
+      "where the record type is the individual data record.",
+    async () => {
+      // Arrange
+      // Queued job.
+      const job = createMock<Job<void>>();
+      mockDownloadFiles(sftpClientMock, [
+        SFAS_INDIVIDUAL_INVALID_RECORDS_FILENAME,
+      ]);
+      // Act
+      const processingResults = await processor.processSFASIntegrationFiles(
+        job,
+      );
+      // Assert
+      const downloadedFile = path.join(
+        process.env.SFAS_RECEIVE_FOLDER,
+        SFAS_INDIVIDUAL_INVALID_RECORDS_FILENAME,
+      );
+      expect(processingResults).toEqual(
+        expect.arrayContaining([
+          expect.objectContaining({
+            summary: expect.arrayContaining([
+              `Processing file ${downloadedFile}.`,
+              "File contains 3 records.",
+              "Error processing record line number 2\nerror: numeric field overflow\ndetail: A field with precision 8, scale 2 must round to an absolute value less than 10^6.",
+              'Error processing record line number 3\nerror: value "4000000000" is out of range for type integer\ndetail: undefined',
+              'Error processing record line number 4\nerror: invalid input syntax for type integer: "NaN"\ndetail: undefined',
```

Comment:
We should never assert runtime errors in tests. As mentioned in the comment https://github.com/bcgov/SIMS/pull/3633/files#r1719016618 process summary should have only user friendly messages which can be asserted.

Reply:
https://github.com/bcgov/SIMS/pull/3633#discussion_r1720046437
The runtime errors in test are to be expected according to the logging implementation. Please confirm we need to revert the logging implementation to its original way.

Reply:
Runtime errors are never something which is expected and also the runtime errors and message can change from time to time based on framework, if the expectation on a record with missing fields is `success: false` and data not saved to the DB we can just assert that.

Reply:
@andrewsignori-aot @guru-aot @bidyashish @katrina-aot-git thoughts??

Reply:
We can refrain from asserting the runtime errors.
Because of the nature of the SFAS bridge and its behavior of stopping the import, we can just check if the errors happen and they will abort the process. The log for this case is just dump the `error` object and we do not need to assert it IMO.